markdown/org/docs/patterns/charlie/instructions/AttachExtensionToRightLeg.svg

```diff
@@ -0,0 +1,347 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!-- Created with Inkscape (http://www.inkscape.org/) -->
+
+<svg
+   width="110.34686mm"
+   height="161.89951mm"
+   viewBox="0 0 110.34686 161.8995"
+   version="1.1"
+   id="svg5908"
+   inkscape:version="1.2.2 (732a01da63, 2022-12-09)"
+   sodipodi:docname="AttachExtensionToRightLeg.svg"
```

Comment:
Export this in inkscape as 'optimized SVG' (or whatever it's called) for a much smaller file size

Reply:
Great I didn't realize this!